leo/commands/node.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the Leo library.
+
+// The Leo library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The Leo library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::commands::{Build, ALEO_CLI_COMMAND};
+use crate::{commands::Command, context::Context};
+use leo_errors::{CliError, PackageError, Result};
+use leo_package::build::BuildDirectory;
+
+use aleo::commands::Node as AleoNode;
+
+use crate::commands::build::BuildOptions;
+use clap::StructOpt;
+use tracing::span::Span;
+
+/// Commands to operate a local development node.
+#[derive(StructOpt, Debug)]
+pub enum Node {
+    /// Starts a local development node
+    Start {
+        /// Skips deploying the local program at genesis.
+        nodeploy: bool,
+    },
+}
+
+impl Command for Node {
+    type Input = <Build as Command>::Output;
+    type Output = ();
+
+    fn log_span(&self) -> Span {
+        tracing::span!(tracing::Level::INFO, "Node")
+    }
+
+    fn prelude(&self, context: Context) -> Result<Self::Input> {
```

Comment:
Does this command need to execute `Build` before running?
On L53, it seems that input is not used.

Reply:
Following up after offline discussion:

The build command will not be run beforehand, an error message will be thrown instead.